include/openssl/bn.h

```diff
@@ -136,14 +136,7 @@ int BN_is_word(const BIGNUM *a, const BN_ULONG w);
 int BN_is_odd(const BIGNUM *a);
 
 # define BN_one(a)       (BN_set_word((a),1))
-
-void BN_zero_ex(BIGNUM *a);
-
-# if OPENSSL_API_COMPAT >= 0x00908000L
-#  define BN_zero(a)      BN_zero_ex(a)
-# else
-#  define BN_zero(a)      (BN_set_word((a),0))
-# endif
+# define BN_zero(a)      (BN_set_word((a),0))
```

Comment:
This seems wrong, you're putting the `BN_zero` implementation back to pre-0.9.8...